server/controllers/userController.js

```diff
@@ -110,21 +110,30 @@ exports.subscriptionChange = async (req, res) => {
   res.redirect('/');
 };
 
+function uncachedCountdown() {
+  return User.count();
+}
+
+/* removed this block of code as we are caching the full server responses now higher up
 function CachedCountDown() {
```

Comment:
this function could likely be completely removed or at least moved into a way where we write a generalised caching function within the caching library.